generators/chipyard/src/main/scala/config/ArianeConfigs.scala

```diff
@@ -13,7 +13,7 @@ class ArianeConfig extends Config(
   new chipyard.config.AbstractConfig)
 
 class dmiArianeConfig extends Config(
-  new chipyard.iobinders.WithTiedOffSerial ++          // Tie off the serial port, override default instantiation of SimSerial
+  new chipyard.harness.WithTiedOffSerial ++          // Tie off the serial port, override default instantiation of SimSerial
```

Comment:
We should be consistent about adding the spaces after the plus. It seems like you missed some.